src/reportGenerator.js

```diff
@@ -70,13 +74,17 @@ class ReportGenerator {
 			const pageTitle = this.config.getPageTitle();
 
 			// Create an xmlbuilder object with HTML and Body tags
+			const useCssFile = this.config.shouldUseCssFile();
```

Comment:
Instead of doing this check here again, add another prop to `renderHtmlReport` called something like 'stylesheetPath' (which is what you pass in in the `generate`-method), which you then always pass down to `createHtmlBase`